### ISSUE TITLE
[Featrure Request] Disable ESLint

### ISSUE THREAD

**Issue by ZimGil** [dderevjanik/wsdl-tsclient#21]
It would be nice if generated files would have a `/* eslint-disable */` line in the top of the file.

Could be options dependent but I guess it could also be default as IMO it would be the standard expected behavior.

**Comment by pfumagalli** [comment]
We just use an `.eslintignore` file containing the directory of the generated types...

https://eslint.org/docs/user-guide/configuring/ignoring-code

**Comment by ZimGil** [comment]
Same here - and yet it could be helpful in some cases.

For example, [ts-proto](https://www.npmjs.com/package/ts-proto) does the same

**Comment by dderevjanik** [comment]
@pfumagalli @ZimGil  Do you think it could be good idea to add an option for this? Or make it default similar to openapi-generator....

**Comment by dderevjanik** [comment]
I did small research regarding `/* eslint-disable*/` in other projects like `openapi-generator` and it seems that some people don't care too much about linting their code and some do. 

check this comment https://github.com/OpenAPITools/openapi-generator/issues/802#issuecomment-542044205 by @[bodograumann](https://github.com/bodograumann) . I think he has the point. Adding path of your generated client to `.eslintignore` is very easy. 

What do you think?

**Comment by ZimGil** [comment]
I agree that the best practice should be to add generated file to `.eslintignore`
Yet I think that it could be nice to have it.

Of course it should be an option and we should not force anything.

As for the default value:
IMHO - most cases would not want to lint generated files.
On the other hand defaulting to `true` would be a breaking change :) so I vote on adding an option with a `false` as default.
